src/network/taraxa_capability.cpp

```diff
@@ -130,499 +187,495 @@ bool TaraxaCapability::interpretCapabilityPacket(NodeID const &_nodeID, unsigned
           }
-          if (pbft_blk_count > 0) {
-            if (syncing_ && peer_syncing_pbft_ == _nodeID) {
-              if (pbft_sync_period_ > pbft_chain_->getPbftChainSize() + (10 * conf_.network_sync_level_size)) {
-                LOG(log_dg_pbft_sync_) << "Syncing pbft blocks faster than processing " << pbft_sync_period_ << " "
-                                       << pbft_chain_->getPbftChainSize();
-                host_.scheduleExecution(1000, [this, _nodeID]() { delayedPbftSync(_nodeID, 1); });
-              } else {
-                syncPeerPbft(_nodeID, pbft_sync_period_ + 1);
-              }
-            } else {
-              LOG(log_dg_pbft_sync_) << "Received PbftBlockPacket from node " << _nodeID
-                                     << " but currently syncing with peer " << peer_syncing_pbft_;
-            }
+        }
+      }
+      if (pbft_blk_count > 0) {
+        if (syncing_ && peer_syncing_pbft_ == _nodeID) {
+          if (pbft_sync_period_ > pbft_chain_->getPbftChainSize() + (10 * conf_.network_sync_level_size)) {
+            LOG(log_dg_pbft_sync_) << "Syncing pbft blocks faster than processing " << pbft_sync_period_ << " "
+                                   << pbft_chain_->getPbftChainSize();
+            util::post(io_service_, 1000, [this, _nodeID] { delayedPbftSync(_nodeID, 1); });
           } else {
-            LOG(log_dg_pbft_sync_) << "Syncing PBFT is completed";
-            // We are pbft synced with the node we are connected to but
-            // calling restartSyncingPbft will check if some nodes have
-            // greater pbft chain size and we should continue syncing with
-            // them
-            restartSyncingPbft(true);
-            // We are pbft synced, send message to other node to start
-            // gossiping new blocks
-            if (!syncing_) {
-              sendSyncedMessage();
-            }
+            syncPeerPbft(_nodeID, pbft_sync_period_ + 1);
           }
-          break;
+        } else {
+          LOG(log_dg_pbft_sync_) << "Received PbftBlockPacket from node " << _nodeID
+                                 << " but currently syncing with peer " << peer_syncing_pbft_;
         }
-        case TestPacket:
-          LOG(log_dg_) << "Received TestPacket";
-          ++cnt_received_messages_[_nodeID];
-          test_sums_[_nodeID] += _r[0].toInt();
-          BOOST_ASSERT(_id == TestPacket);
-          return (_id == TestPacket);
-      };
+      } else {
+        LOG(log_dg_pbft_sync_) << "Syncing PBFT is completed";
+        // We are pbft synced with the node we are connected to but
+        // calling restartSyncingPbft will check if some nodes have
+        // greater pbft chain size and we should continue syncing with
+        // them
+        restartSyncingPbft(true);
+        // We are pbft synced, send message to other node to start
+        // gossiping new blocks
+        if (!syncing_) {
+          sendSyncedMessage();
```

Comment:
The synced message will clean up peer clearAllKnownBlocksAndTransactions.
Why need to clear all DAG blocks and transactions?
This is inside PbftBlockPacket. Why don't clear PBFT blocks and votes?

Reply:
I think this change is obsolete, I never wrote that code, perhaps this was displayed as a diff because at some point this branch diverged from develop